run/shared/kpp_standalone_interface.yml

```diff
@@ -0,0 +1,77 @@
+active_cells:
```

Comment:
@obin1, I just have a couple of thoughts.

1. Right now this file isn't automatically copied when we create a run directory.  That's an easy fix though, we'd just have to edit `run/GCClassic/createRunDir.sh` accordingly.
2. I think it'd be nice to have a master on/off switch either in this file or in `geoschem_config.yml`.  If we copy this file to the run directory, right now there is no way to disable saving this output (which most users probably won't want to do.)   
Interested to hear your thoughts.

Reply:
Hi @yantosca, sorry for missing this earlier! 
Regarding point 1, I've never created a GEOS-Chem run directory, we've just been copying prior GEOS-CF run directories. Automatically copying sounds like the way to go.
For point 2: If I understand some subsequent commits, you've made that on/off switch. That is better than what I've been doing, which is just renaming the file when I didn't want to print output.